packages/components/pages/f-registration/test-utils/component-objects/f-registration.component.js

```diff
@@ -82,47 +82,47 @@ module.exports = class Registration extends Page {
     };
 
 
-    load () {
-        super.load(this.component);
+    async load () {
+        await super.load(this.component);
     }
 
-    waitForComponent () {
-        super.waitForComponent(this.component);
+    async waitForComponent () {
+        await super.waitForComponent(this.component);
     }
 
-    isComponentDisplayed () {
+    async isComponentDisplayed () {
         return this.component.isDisplayed();
     }
 
-    isInputFieldDisplayed (fieldName) {
+    async isInputFieldDisplayed (fieldName) {
         return this.fields[fieldName].input.isDisplayed();
     }
 
-    submit () {
-        this.createAccountButton.click();
+    async submit () {
+        await this.createAccountButton.click();
     }
 
-    isEmptyErrorDisplayed (fieldName) {
+    async isEmptyErrorDisplayed (fieldName) {
```

Comment:
Async but no Await ... don't think this will do anything ... plus it is literally just setting variables.